### ISSUE TITLE
Archive this repository once segmented behavior is merged into clamsproject/app-puakaldi-wrapper

### ISSUE THREAD

**Issue by angus-lherrou** [clamsproject/app-puakaldi-segmented-wrapper#1]
If and when clamsproject/app-puakaldi-wrapper#10 is merged and clamsproject/app-puakaldi-wrapper#1 is closed, this repository can be archived.
